bin/merge_script

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
```

Comment:
I think it should have a license header.
Even if it is "just a shell script", I think one would be better safe than sorry by adding it.

Reply:
True!